Source/Laser/LaserProfilesImpl/LaserProfileFromTXYEFile.cpp

```diff
@@ -130,35 +140,25 @@ WarpXLaserProfiles::FromTXYEFileLaserProfile::fill_amplitude (
         Abort("Something bad has happened with the simulation time");
     }
 
-    if(m_params.is_grid_uniform){
-        internal_fill_amplitude_uniform(
-            idx_t_left, np, Xp, Yp, t, amplitude);
-    }
-    else{
-        internal_fill_amplitude_nonuniform(
-            idx_t_left, np, Xp, Yp, t, amplitude);
-    }
+    internal_fill_amplitude_uniform(idx_t_left, np, Xp, Yp, t, amplitude);
 }
 
 void
 WarpXLaserProfiles::FromTXYEFileLaserProfile::parse_txye_file(std::string txye_file_name)
 {
+
     if(ParallelDescriptor::IOProcessor()){
-        std::ifstream inp(txye_file_name, std::ios::binary);
-        if(!inp) Abort("Failed to open txye file");
-        inp.exceptions(std::ios_base::failbit | std::ios_base::badbit);
-
-        //Uniform grid flag
-        char flag;
-        inp.read(&flag, 1);
-        if(!inp) Abort("Failed to read grid type from txye file");
-        m_params.is_grid_uniform=flag;
-
-        //Grid points along t, x and y
-        inp.read(reinterpret_cast<char*>(&m_params.nt), sizeof(uint32_t));
-        inp.read(reinterpret_cast<char*>(&m_params.nx), sizeof(uint32_t));
-        inp.read(reinterpret_cast<char*>(&m_params.ny), sizeof(uint32_t));
-        if(!inp) Abort("Failed to read sizes from txye file");
+
+        auto series = io::Series(txye_file_name, io::Access::READ_ONLY);
+        auto i = series.iterations[0];
+        auto E = i.meshes["E_real"];
```

Comment:
```suggestion
        auto E = i.meshes["laserEnvelope"];
```
Since this was recently changed in `lasy`: https://github.com/LASY-org/lasy/blob/development/lasy/utils/openpmd_output.py#L49